app/gui2/src/util/ast/index.ts

```diff
@@ -99,14 +99,13 @@ export function* astContainingChar(charIndex: number, root: Tree): Generator<Tre
 if (import.meta.vitest) {
   const { test, expect } = import.meta.vitest
 
-  // Not working cases commented.
   const parseCases = [
-    ' foo bar\n',
```

Comment:
I think that space was intentional.

Reply:
It was, but I've changed handling of this case. It is now a syntax error to start a file with an indented line. This is now covered by the parser unit tests.